.github/workflows/bump-beats-version.yml

```diff
@@ -0,0 +1,65 @@
+name: update-beats
+
+on:
+  workflow_dispatch:
+  schedule:
+  - cron: '0 0 * * 1-5'
+
+permissions:
+  contents: read
+
+jobs:
+  filter:
+    runs-on: ubuntu-latest
+    timeout-minutes: 1
+    outputs:
+      matrix: ${{ steps.generator.outputs.matrix }}
+    permissions:
+      contents: read
+    steps:
+    - id: generator
+      uses: elastic/oblt-actions/elastic/active-branches@v1
+      with:
+        exclude-branches: "7.17,8.17,8.18,9.0"
+        filter-branches: true
+
+  update-beats:
+    permissions:
+      contents: write
+      pull-requests: write
+    needs:
+    - filter
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix: ${{ fromJson(needs.filter.outputs.matrix) }}
+    steps:
+    - uses: actions/checkout@v4
+      with:
+        ref: ${{ matrix.branch }}
```

Comment:
> It's main and 8.19. We will probably enable this for 9.0 at some point in the future as well.

If so, you need to backport to the relevant branches, otherwise `ref` will try to run the updatecli pipeline and scripts from those branches but won't be available.

Reply:
I've just added those labels:

- https://github.com/elastic/elastic-agent/pull/8174#event-17749902545

Reply:
My understanding is that the updatecli script needs to be backported, but the GitHub action definition should not be backported. Is this correct?

Reply:
Scheduled actions only run from the default branch, so this doesn't make any difference. I'd backport everything just to keep things consistent.

Reply:
> but the GitHub action definition should not be backported. Is this correct?

Yes